Sources/PrimerSDK/Classes/User Interface/Apple Pay/ApplePayViewModel.swift

```diff
@@ -57,19 +57,31 @@ class ApplePayViewModel: NSObject, ApplePayViewModelProtocol {
     // swiftlint:disable cyclomatic_complexity function_body_length
     func payWithApple(completion: @escaping (Error?) -> Void) {
         guard let countryCode = countryCode else {
-            return completion(PaymentException.missingCountryCode)
+            let err = PaymentException.missingCountryCode
```

Comment:
[Suggestion]
Loads of duplicated code here. Would it be possible to put the common stuff in some function?